enterprise/coderd/groups.go

```diff
@@ -156,7 +156,7 @@ func (api *API) patchGroup(rw http.ResponseWriter, r *http.Request) {
 
 	currentMembers, err := api.Database.GetGroupMembersByGroupID(ctx, database.GetGroupMembersByGroupIDParams{
 		GroupID:       group.ID,
-		IncludeSystem: false,
+		IncludeSystem: true,
```

Comment:
Do we still need all these `IncludeSystem: true` cases?

Reply:
@mtojek would you be able to provide consensus on this?